### PR TITLE
Upgrade tc-lib-api

### PR DESCRIPTION
**Pull request by fiennyangeln**
-Upgrade taskcluster-lib-api to carry Access-Control-Max-Age
-Add space after comma in main.js to pass eslint

Related to Bug 1442636

@djmitche please review:D

### CODE REVIEW

**Comment by djmitche** [comment]
It looks like you accidentally ran `git add` on `lib/`?  Lots of changes here!

**Comment by fiennyangeln** [comment]
@djmitche okay I think I fixed it :D I need to be more careful later when using `git add .`

File: src/api-v1.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ let GENERIC_ID_PATTERN = /^[a-zA-Z0-9-_]{1,22}$/;
 let api = new API({
   title: 'Cloud Mirror API',
   description: 'Service to duplicate URLs from various cloud providers',
+  name: 'cloudmirror',
```
Comment (djmitche) [inline src/api-v1.js]: This should be `cloud-mirror`.  The `-` is allowed as long as it's not the first charcter:
https://github.com/taskcluster/taskcluster-lib-api/blob/master/src/api.js#L650

**Comment by fiennyangeln** [comment]
@djmitche changed!:D thanks for the guides

**Review by jhford — APPROVED** [review]
Looks good to me, but I'm not sure when we'll get a chance to deploy this fix.

**Comment by jhford** [comment]
Closing as we'll pursue the object service instead of updating cloud-mirror

**Pull request closed without merging**